scripts/collect_steps.py

```diff
@@ -165,14 +155,39 @@ def ipa_steps():
                     SECURITY_MODEL,
                     "-t",
                     str(tv),
-            ]
+                ]
             print(" ".join(args), file=sys.stderr)
             output.update(collect_steps(args))
     return output
 
+
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generate steps file")
+    parser.add_argument(
+        "-m",
+        "--multi-threading",
+        action="store_true",
+    )
+    args = parser.parse_args()
+
+    features = "enable-benches debug-trace step-trace"
+    if args.multi_threading:
+        features = f"{features} multi-threading"
```

Comment:
Shouldn't this use a list and `" ".join()`?

Reply:
if we wanted to make this more flexible (e.g., add other flags too), that's how I'd do it. @akoshelev suggested the steps concept was going away soon, so I just did this quick an dirty.

Reply:
we are probably still going to use steps, but this script should go away after #961